scipy/special/add_newdocs.py

```diff
@@ -2768,30 +2768,67 @@ def add_newdoc(name, doc):
 
     Regularized lower incomplete gamma function.
 
-    Defined as
+    It is defined as
 
     .. math::
 
-        \frac{1}{\Gamma(a)} \int_0^x t^{a - 1}e^{-t} dt
+        P(a, x) = \frac{1}{\Gamma(a)} \int_0^x t^{a - 1}e^{-t} dt
 
-    for :math:`a > 0` and :math:`x \geq 0`. The function satisfies the
-    relation ``gammainc(a, x) + gammaincc(a, x) = 1`` where
-    `gammaincc` is the regularized upper incomplete gamma function.
+    for :math:`a > 0` and :math:`x \geq 0`. See [1]_ for details.
+
+    Parameters
+    ----------
+    a : array_like
+        Positive parameter
+    x : array_like
+        Nonnegative argument
+
+    Returns
+    -------
+    scalar or ndarray
+        Values of the lower incomplete gamma function
 
     Notes
     -----
-    The implementation largely follows that of [1]_.
+    The function satisfies the relation ``gammainc(a, x) +
+    gammaincc(a, x) = 1`` where `gammaincc` is the regularized upper
+    incomplete gamma function.
+
+    The implementation largely follows that of [2]_.
 
     See also
     --------
     gammaincc : regularized upper incomplete gamma function
-    gammaincinv : inverse to ``gammainc`` versus ``x``
-    gammainccinv : inverse to ``gammaincc`` versus ``x``
+    gammaincinv : inverse of the regularized lower incomplete gamma
+        function with respect to `x`
+    gammainccinv : inverse of the regularized upper incomplete gamma
+        function with respect to `x`
 
     References
     ----------
-    .. [1] Maddock et. al., "Incomplete Gamma Functions",
+    .. [1] NIST Digital Library of Mathematical functions
```

Comment:
Most (but not all) of the references to DLMF in this file use `[dlmf]`  E.g.
```
    References
    ----------
    .. [dlmf] NIST Digital Library of Mathematical Functions
              https://dlmf.nist.gov/5.2#E1
```

Using `[1]_` `[2]_`,...  is more consistent within each function body documentation, but it requires scrolling down to the References to see what is being referenced.  On the other hand, DLMF is the (2nd) most common reference in `add_newdocs.py`,  so that whenever I see `[dlmf]` in the doc, I know immediately what it is referring to without having to look any further.  
Is there a standard to apply going forward? (I can see arguments in favor of both approaches.)

Reply:
I’ve generally been using dlmf, but then it looked weird when the other reference was a number. Maybe I should have instead made the other reference non-numerical? Not sure...

Reply:
Went ahead with using `[dlmf]`, and changes the other one to `[boost]`, which also seemed more descriptive.

Reply:
I think numpydoc wanted the numbered syntax.